src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,48 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamNumber(2, 4) || !L.CheckParamSelf("cColor"))
+	{
+		return 0;
+	}
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
I was having a bit of trouble with that:

```
cColor * self;
unsigned char r, g, b;
if (!L.GetStackValues(1, self, r, g, b)) {
   ...
}
```

Throws a giant error message of the form:

```
/home/lane/minecraft/cuberite/src/Bindings/LuaState.h:663:7: error: no type named ‘type’ in ‘struct std::enable_if<false, void>’
/home/lane/minecraft/cuberite/src/Bindings/LuaState.h: In instantiation of ‘bool cLuaState::GetStackValues(int, Arg1&&, Args&& ...) [with Arg1 = cColor*&; Args = {unsigned char&, unsigned char&, unsigned char&}]’:
/home/lane/minecraft/cuberite/src/Bindings/ManualBindings.cpp:3698:40:   required from here
/home/lane/minecraft/cuberite/src/Bindings/LuaState.h:685:7: note: candidate: template<class T> bool cLuaState::GetStackValue(int, cLuaState::cOptionalParam<T>&&)
  bool GetStackValue(int a_StackPos, cOptionalParam<T> && a_ReturnedVal)
       ^~~~~~~~~~~~~
/home/lane/minecraft/cuberite/src/Bindings/LuaState.h:685:7: note:   template argument deduction/substitution failed:
In file included from /home/lane/minecraft/cuberite/src/Bindings/ManualBindings.h:12:0,
                 from /home/lane/minecraft/cuberite/src/Bindings/ManualBindings.cpp:4:
/home/lane/minecraft/cuberite/src/Bindings/LuaState.h:757:7: note:   mismatched types ‘cLuaState::cOptionalParam<T>’ and ‘cColor*’
   if (!GetStackValue(a_StartStackPos, std::forward<Arg1>(a_Arg1)))
        
In file included from /home/lane/minecraft/cuberite/src/Bindings/LuaState.h:733:0,
                 from /home/lane/minecraft/cuberite/src/Bindings/ManualBindings.h:12,
                 from /home/lane/minecraft/cuberite/src/Bindings/ManualBindings.cpp:4:
/home/lane/minecraft/cuberite/src/Bindings/LuaState_Declaration.inc:94:6: note: candidate: bool cLuaState::GetStackValue(int, cArrowEntity*&)
 bool GetStackValue(int a_StackPos, PtrcArrowEntity & a_ReturnedVal);
      ^~~~~~~~~~~~~
/home/lane/minecraft/cuberite/src/Bindings/LuaState_Declaration.inc:94:6: note:   no known conversion for argument 2 from ‘cColor*’ to ‘cArrowEntity*&’
/home/lane/minecraft/cuberite/src/Bindings/LuaState_Declaration.inc:95:6: note: candidate: bool cLuaState::GetStackValue(int, const cArrowEntity*&)
 bool GetStackValue(int a_StackPos, ConstPtrcArrowEntity & a_ReturnedVal);
      ^~~~~~~~~~~~~
/home/lane/minecraft/cuberite/src/Bindings/LuaState_Declaration.inc:95:6: note:   no known conversion for argument 2 from ‘cColor*’ to ‘const cArrowEntity*&’
/home/lane/minecraft/cuberite/src/Bindings/LuaState_Declaration.inc:96:6: note: candidate: bool cLuaState::GetStackValue(int, cBeaconEntity*&)
 bool GetStackValue(int a_StackPos, PtrcBeaconEntity & a_ReturnedVal);
      ^~~~~~~~~~~~~
```
Which continues for several hundred different types. I suspect this has to do with the fact that cColor isn't exported by tolua? (since it seems to be matching it against every class that is exported by tolua)

Other than this, I've alpha sorted everything.